scss/_patterns_lists.scss

```diff
@@ -127,41 +165,47 @@
   }
 }
 
-// The placeholder for basic border bottom lists
+// Placeholder for basic styled lists
 %list {
   list-style: none;
   margin-left: 0;
+  padding-left: 0;
 
-  li {
-    border-bottom: 1px dotted $color-warm-grey;
-    margin-bottom: 0;
-    padding: 10px 0;
+  &__item {
+    @extend %list-item;
   }
+}
 
-  li:last-of-type,
+// Placeholder for list items
+%list-item {
```

Comment:
It looks like the standard that we're using is that all placeholder names just use a single "-"